api/applications/applicationRouter.js

```diff
@@ -1,16 +1,14 @@
 const express = require('express');
 // const authRequired = require('../middleware/authRequired');
-const Application = require('./applicationModel');
+//const Application = require('./applicationModel');
```

Comment:
@ashurik522 @matt-arofin Can you please remove any comments that aren't strictly necessary?